integration-tests/cucumber-tests-platform-microgrids/src/test/java/org/opensmartgridplatform/cucumber/platform/microgrids/database/Database.java

```diff
@@ -45,12 +45,14 @@ private void insertDefaultData() {
     private List<NotificationWebServiceConfiguration> notificationEndpointConfigurations() {
         final NotificationWebServiceConfigurationBuilder builder = new NotificationWebServiceConfigurationBuilder()
                 .withApplicationName("ZownStream")
-                .withMarshallerContextPath("org.opensmartgridplatform.adapter.ws.schema.microgrids.notification");
+                .withMarshallerContextPath("org.opensmartgridplatform.adapter.ws.schema.microgrids.notification")
+                .withoutCircuitBreakerConfig();
```

Comment:
What effect has the addition of `.withoutCircuitBreakerConfig()` for the test-code? Are the default circuit breaker values somehow causing problems?

Reply:
Yes, the circuit breaker causes problems for the test scenario. The circuit breaker prevents the return of a Notification the test scenario is waiting for. WIth `.withoutCircuitBreakerConfig()` the circuit breaker is not used for all Microgrids test scenarios.